test/issues/0519.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var assert = require('assert');
+var yaml   = require('../../');
+
+test('Loader should not strip quotes for lines containing equals sign', function () {
```

Comment:
I'm a bit confused about test name. If problem is in dumper (missed quotes for `=`), why title is bout testing loader?

Reply:
Thanks for catching that! I update the name to be more clear.